src/main/java/com/questhelper/helpers/quests/templeofikov/TempleOfIkov.java

```diff
@@ -344,7 +344,7 @@ public void setupSteps()
 		makeChoice.addDialogStep("Ok! I'll help!");
 
 		killLucien = new NpcStep(this, NpcID.LUCIEN, new WorldPoint(3122, 3484, 0), "Equip the Armadyl Pendant and kill Lucien in the house west of the Grand Exchange.", armadylPendant);
-		bringStaffToLucien = new NpcStep(this, NpcID.LUCIEN, new WorldPoint(3122, 3484, 0), "Bring the Staff of Armadyl to Lucien in the house west of the Grand Exchange.", staffOfArmadyl);
+		bringStaffToLucien = new NpcStep(this, NpcID.LUCIEN_13608, new WorldPoint(3122, 3484, 0), "Bring the Staff of Armadyl to Lucien in the house west of the Grand Exchange.", staffOfArmadyl);
```

Comment:
Since we're not sure exactly of the circumstances for when which NPC ID will be relevant (the old ID was correct at some point), it'd be best to add this new ID you found as an alternative ID

```suggestion
		bringStaffToLucien = new NpcStep(this, NpcID.LUCIEN_13608, new WorldPoint(3122, 3484, 0), "Bring the Staff of Armadyl to Lucien in the house west of the Grand Exchange.", staffOfArmadyl);
		((NpcStep) bringStaffToLucien).addAlternateNpcs(NpcID.LUCIEN_13608);
```

Reply:
Good point, I'll do this!